crates/iota-sdk/README.md

```diff
@@ -27,6 +27,10 @@ async fn main() -> Result<(), anyhow::Error> {
     let iota_devnet = IotaClientBuilder::default().build_devnet().await?;
     println!("Iota devnet version: {}", iota_devnet.api_version());
 
+    // Iota mainnet -- https://api.mainnet.iota.cafe
```

Comment:
Just a small nitpick, we should try to write `IOTA` everywhere, only where it is explicitly camel case we can use `Iota`.
Do you mind updating all 3 comments?

Reply:
Yep, no problem!